src/rest-server/src/models/v2/job/k8s.js

```diff
@@ -1142,6 +1186,63 @@ const getSshInfo = async (frameworkName) => {
   );
 };
 
+const addTag = async (frameworkName, tag) => {
+  // check if frameworkName exist
+  const framework = await databaseModel.Framework.findOne({
+    where: { name: encodeName(frameworkName) },
+  });
+
+  if (framework) {
+    // add tag
+    const data = await databaseModel.Tag.findOrCreate({
+      where: {
+        frameworkName: encodeName(frameworkName),
+        name: tag,
+        uid: encodeName(frameworkName + tag),
```

Comment:
`encodeName(frameworkName + "#%" + tag)` will be better?

Reply:
> `encodeName(frameworkName + "#%" + tag)` will be better?

Using encodeName(frameworkName + "+" + tag) to be consistent with the encoding algo of framework_history uid.